test/metabase/query_processor/preprocess_test.clj

```diff
@@ -29,9 +31,6 @@
               (is (= {:cached?  false
                       :num-rows 5}
                      (run-query))))
-            ;; run a few more times to make sure stuff got a chance to be cached.
-            (run-query)
-            (run-query)
```

Comment:
What was going on here? Glad it's not needed anymore 😅

Reply:
Logic was a bit broken, cache was stored on 2nd call and used only on 3rd.

Reply:
Glad that's fixed!